java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -203,7 +203,7 @@ private ScheduledExecutorService getSchedulerOrDefault() {
     private String createBody() {
       var body = new StringBuilder("grant_type=").append(grantType);
       if (!scopes.isEmpty()) {
-        body.append("&").append("scopes=").append(String.join(",", scopes));
+        body.append("&").append("scope=").append(String.join("%20", scopes));
```

Comment:
This is just a bug fix right?  Wonder how I got this so wrong 🤔

Reply:
Yeah, the URL was malformed and resulted in a wrong access token (without any scopes or audience field). Happens to all of us :D  there's also small changes in the BankIDScopes in platform PR.

Reply:
Better to URLEncode the final scope value (see my comment on platform PR also).